### PR TITLE
created new test class

### PR DESCRIPTION
**Pull request by vmlysenkov**

### CODE REVIEW

**Review by khodya — APPROVED** [review]

**Comment by vmlysenkov** [comment]
@khodya please, check

File: src/main/java/DetermineCourseCompletion.java [hunk 1]
```diff
@@ -43,4 +44,24 @@ public static LocalDateTime getCourseCompletionDate(LocalDateTime startDate, int
         }
         return endDate;
     }
+
+    public static LocalDateTime determineEndDateSkippingWeekends(LocalDateTime startDate, int coursesDuration) {
```
Comment (khodya) [inline src/main/java/DetermineCourseCompletion.java:48]: could you implement this logic inside the `getCourseCompletionDate` method? 
thus, we ensure existing tests are passing for `getCourseCompletionDate`

File: src/test/java/DetermineCourseCompletionTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import static org.junit.Assert.*;
+
+public class DetermineCourseCompletionTest {
+
+    @Test
+    public void shouldDetermineEndDateWhenCourseEndsAt6PM() {
+        LocalDateTime startDate = LocalDateTime.of(2022, Month.FEBRUARY, 4, 10, 0);
+        int coursesDuration = 64;
+
+        LocalDateTime actual = DetermineCourseCompletion.determineEndDateSkippingWeekends(startDate, coursesDuration);
```
Comment (khodya) [inline src/test/java/DetermineCourseCompletionTest.java:15]: Could you make the tests working against `DetermineCourseCompletion.getCourseCompletionDate` method? 
thanks

**Review by khodya — COMMENTED** [review]
Changes required

**Pull request merged**